### PR TITLE
Apply example_ynh and upgrade to 10.8.1

### PR DESCRIPTION
**Pull request by yalh76**
## Solution

- *Apply example_ynh*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
@tituspijean  do we merge or do we wait for #108 ?

**Comment by tituspijean** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -129,6 +152,13 @@ ynh_script_progression --message="Restoring the systemd configuration..." --weig
 ynh_restore_file --origin_path="/etc/systemd/system/jellyfin.service.d" --not_mandatory
 systemctl enable jellyfin.service --quiet
 
+#=================================================
+# RESTORE THE LOGROTATE CONFIGURATION
+#=================================================
+ynh_script_progression --message="Restoring the logrotate configuration..." --time --weight=1
```
Comment (tituspijean) [inline scripts/restore:158]:
```suggestion
ynh_script_progression --message="Restoring the logrotate configuration..." --weight=1
```

**Review by tituspijean — CHANGES_REQUESTED** [review]

**Review by tituspijean — APPROVED** [review]

**Pull request merged**